custom/panel_templates/Default/integrations/discord/discord.tpl

```diff
@@ -39,7 +39,6 @@
                             <!-- Success and Error Alerts -->
                             {include file='includes/alerts.tpl'}
 
-                            <form action="" method="POST">
```

Comment:
Can we un-indent the rest of this section?

Reply:
What do you mean?

Reply:
Done 👍